firedrills.md

```diff
@@ -168,6 +192,40 @@ node-1-planet$ journalctl -u kube-apiserver --no-pager
 node-1-planet$ journalctl -u kube-kubelet --no-pager
 ```
 
+Most of these services run with standard logging settings (usually logging at INFO level) and you may want to increase the verbosity of the log output of a certain component when troubleshooting something. Different processes have different ways of increasing the log verbosity, Kubernetes services usually take a `-v` flag followed by a numeric verbosity level, 1-10.
+
+Let's see how we can get kubelet to log more information.
+
+First, we need to identify where its systemd unit file is located as we'll need to edit it to add the flag. Depending on the service, its unit file may be found in `/lib/systemd/system` or `/etc/systemd/system` so to know for sure let's check the unit status:
+
+```bash
+node-1-planet$ systemctl status kube-kubelet
+● kube-kubelet.service - Kubernetes Kubelet
+   Loaded: loaded (/lib/systemd/system/kube-kubelet.service; static; vendor preset: enabled)
+   Active: active (running) since Mon 2019-10-07 19:19:13 UTC; 3 months 9 days ago
+```
+
+Once the unit's service spec file is identified, edit it and add a `-v5` flag somewhere to the `ExecStart` command:
+
+```bash
+node-1-planet$ nano /lib/systemd/system/kube-kubelet.service
```

Comment:
I think the proper way of making changes to systemd units is [`systemctl edit`](https://www.freedesktop.org/software/systemd/man/systemctl.html#edit%20UNIT%E2%80%A6) - i.e. `systemctl edit --full service` which will create drop-ins in the corresponding directory and auto-reload after making changes.

Reply:
Added this too.